### PR TITLE
ignore: triggering tests

### PR DESCRIPTION
**Pull request by olevski**
I am not really sure that is is enough to make sure the tests run with git LFS version 3.2.0 (i..e the latest)

@Panaetius do you know how I can check or make sure that this is the case?

### CODE REVIEW

**Comment by Panaetius** [comment]
Just append `&& git lfs --version` and check what it prints?

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ FROM python:3.9-slim as base
 
 # hadolint ignore=DL3008,DL3009,DL3013
 RUN apt-get update && \
-    apt-get install --no-install-recommends -y git git-lfs=2.* python3-dev && \
+    apt-get install --no-install-recommends -y git git-lfs=2.3.0 python3-dev && \
```
Comment (Panaetius) [inline Dockerfile:5]:
```suggestion
    apt-get install --no-install-recommends -y git git-lfs=3.2.0 python3-dev && \
```
I think?

**Comment by Panaetius** [comment]
We don't use the Dockerfile in our tests, so you need to change all https://github.com/SwissDataScienceCenter/renku-python/blob/develop/.github/workflows/test_deploy.yml#L532 lines as well. I think the linux tests default to whatever comes from github, which should be a new version

**Comment by Panaetius** [comment]
By the way, closes https://github.com/SwissDataScienceCenter/renku-python/issues/2945 ?

**Comment by olevski** [comment]
This is not needed anymore. It was only used to trigger the tests.

**Pull request closed without merging**